src/verify.integration.test.ts

```diff
@@ -152,3 +152,174 @@ describe("verify(integration)", () => {
     );
   });
 });
+
+describe("verifyWithIndividualChecks(integration)", () => {
+  it("returns false if document is invalid", async () => {
```

Comment:
I think `document is invalid` is not clear enough on the intent, From the expectation:
- document has not been issued
- hash is invalid

What are we testing in that case ?